integration-tests/src/main/scala/com/typesafe/sbtrc/it/loading/CanLoadSimpleProject.scala

```diff
@@ -173,33 +168,6 @@ class CanLoadSimpleProject extends SbtClientTest {
       case _ =>
     })(keepEventsInOrderExecutor)
 
-    def withCompileTaskResult(body: Future[Analysis] => Unit): Unit = {
-      val result = Promise[Analysis]
-      val compileWatchSub: Subscription = (client.rawWatch(TaskKey[Analysis](compileKeys.head)) { (a: ScopedKey, b: TaskResult) =>
-        result.tryComplete(b.resultWithCustomThrowable[Analysis, CompileFailedException])
-      })(keepEventsInOrderExecutor)
-
-      try body(result.future)
-      finally compileWatchSub.cancel()
-    }
-    withCompileTaskResult { compileWatchFuture =>
-      client.requestExecution("compile", None)
-      val gotException =
-        try {
-          waitWithError(compileWatchFuture, "Unable get compile analysis from server")
-          false
-        } catch {
-          case e: CompileFailedException =>
-            if (e.problems.isEmpty)
-              throw new AssertionError(s"CompileFailedException had no problems in it $e")
-            true
-          case e: Throwable =>
-            throw new AssertionError(s"expected CompileFailedException, got $e")
-            true
-        }
-      if (!gotException)
-        throw new AssertionError(s"Expected compile to fail but it didn't")
```

Comment:
Scratch that.  All compilation failures for Play are handled server-side.  Can dump it.